web/tests/stream_search.test.cjs

```diff
@@ -135,10 +135,6 @@ run_test("basics", ({override, override_rewire}) => {
     verify_expanded();
     verify_focused();
 
-    // Clear an empty search.
-    clear_search_input();
-    verify_collapsed();
```

Comment:
Why isn't this still worth testing?

Reply:
The user won't be able to clear an already empty channel filter since we don't show the "clear" button when the input is empty. That was my thought while removing this part of the test.

Reply:
But there are other code paths that call that function when it's empty, not just the button, right?